src/target/llvm/codegen_hexagon.cc

```diff
@@ -76,6 +76,8 @@ class CodeGenHexagon final : public CodeGenLLVM {
   llvm::FunctionType* ftype_tvm_api_set_last_error_{nullptr};
 
  private:
+  TypedPointer CreateBufferPtr(llvm::Value* buffer_ptr, DataType buffer_element_dtype,
+                               std::vector<llvm::Value*> indices, DataType value_dtype) final;
```

Comment:
You can use `llvm::ArrayRef<llvm::Value*>` instead of `std::vector`, it's a bit more lightweight structure for non-modifiable arrays.